.github/workflows/deploy-prerelease.yml

```diff
@@ -7,6 +7,7 @@ on:
       - 'config/**'
       - 'modules/**'
       - 'env-dev.yml'
+      - '.github/workflows/deploy-prerelease.yml'
```

Comment:
Why on modification to a `.github/*` file?

Reply:
If we change how a release is deployed (by changing the associated workflow) I think we should deploy a new release with the "updated" method. 
It is true though we might not want that every time.
I think we have two options:
- default to a new release deployment trigger that we can prevent using `[skip ci]` in the commit message
- not trigger when the workflow changes, but then give the possibility to trigger a release in case we want (for example by adding a `workflow_dispatch` event trigger

I went for the first option, but the second one might be safer (in case people don't remember to use `[skip ci]` when they don't want a new release.

Reply:
I like option 2 - I don't like the idea of CI changes triggering builds. Can you do a `repository_dispatch` event manually?

Reply:
"Manually" meaning by the click of a button on GitHub I don't think so. 
You can use `gh`, but anyway there are currently some jobs that get executed only if the trigger was a `repository_dispatch` event, because it considers that done by another workflow. 
To avoid having to change too many things, I would simply add the `workflow_dispatch` trigger option (it would also make it easier to trigger)

Reply:
Fixed in [857f983b9206c20f187a9058444f54477b8f6807](https://github.com/ACCESS-NRI/access-ram-condaenv/pull/13/commits/857f983b9206c20f187a9058444f54477b8f6807)